packages/storage/src/core/storage.ts

```diff
@@ -206,7 +206,7 @@ export class ThirdwebStorage<T extends UploadOptions = IpfsUploadBatchOptions> {
 
     if (files.length) {
       // Upload all files that came from the object
-      const uris = await this.uploader.uploadBatch(files);
```

Comment:
oof.. make sure that all other calls pass the options properly

Reply:
This one is tricky - the way we used to do it is only pass through options (and onProgress) through to one of the calls on `batchUpload` because otherwise the progress would reset in the middle (since we upload files first, then the top-level object). So we may want to only pass through onProgress to the file upload.

Can keep this as a fix for now though